CQUI.modinfo

```diff
@@ -264,11 +264,16 @@
         <LocalizedText id="CQUI_TEXT">
             <Items>
                 <File>Assets/Text/Gossip_Text.xml</File>
+                <File>Assets/Text/Gossip_Text_de.xml</File>
```

Comment:
What's up with these new "it" and de file references?

Reply:
it appears those files exist, (Gossip_Text_de.xml and Gossip_Text_it.xml) but aren't currently referenced by the modinfo?

There is an error in the Gossip_Text_de.xml that is easily fixed: 
Line 4 has one too many dashes, between "Text" and "GERMAN", it's confusing for XML parsers
```xml
		<!-- Replace Gossip Text -- GERMAN -->
```
Change that to <!-- Replace Gossip Text - German -->

Reply:
Yes, those files exist, but aren't currently referenced by the modinfo.
I also fixed an error in the comment.